addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- RepeaterMode channel -->
+	<channel-type id="repeaterMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Repeater Mode</label>
+		<description>Repeater mode of EnOcean gateway/device</description>
+		<state>
+			<options>
+				<option value="OFF">Repeater off</option>
+				<option value="LEVEL1">Level-1</option>
+				<option value="LEVEL2">Level-2</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="setBaseId" advanced="true">
```

Comment:
Sounds like a candidate for https://github.com/eclipse/smarthome/issues/3484 instead of being a channel.

Reply:
Yes you are absolutely right. However as you can only change this BaseId 10 times, I wanted to make this fool proof and setup an advanced channel for that. In this way the user explicitly has to link and interact with it.

Reply:
ok, sounds like the safest solution then.